### DISCUSSION TITLE
Using Orbit with Isaac Sim Cortex Framework

### DISCUSSION THREAD

**Discussion by andreipit** [isaac-sim/IsaacLab#207] [Q&A]
How to use Cortex in Orbit?
They have different world classes:
1) Orbit:
```
from omni.isaac.orbit.sim import SimulationContext
sim = SimulationContext(sim_cfg)
sim.step()
```
2) Cortex:
```
from omni.isaac.cortex.cortex_world import CortexWorld
world = CortexWorld()
world.step()
```
Why Orbit does not use Cortex in examples? 
Are there any plans to use Cortex in Orbit?

**Comment by eferreirafilho** [comment]
I believe this will come in future releases:

https://forums.developer.nvidia.com/t/integrate-isaac-gym-in-isaac-cortex/272874
